src/firewall/server/dispatcher.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import os
+import subprocess
+
+from firewall.core.logger import log
+from firewall.config import FIREWALLD_DISPATCHER, ETC_FIREWALLD_DISPATCHER, COMMANDS
+
+
+def run_dispatcher(interface, signal, args):
+    def _dispatchers(directories):
+        for directory in directories:
+            if not os.path.isdir(directory):
+                continue
+            for file in os.listdir(directory):
+                _path = os.path.abspath(directory + os.sep + file)
+                if os.path.isfile(_path) and os.access(_path, os.X_OK):
+                    yield _path
+
+    env = {"LANG": "C"}
```

Comment:
doesn't `systemd-run` sanitize the environment? I think we would want that...

I think, if we want to set environment variables, we need to pass them as `-E` arguments to `systemd-run`.


systemd-run ends up logging lines like
```
# journalctl -u run-ref10f3923f304b379bfcf732e79d2987.service
Jan 19 15:48:07 rh1 systemd[1]: Started run-ref10f3923f304b379bfcf732e79d2987.service - /usr/bin/echo hallo world.
Jan 19 15:48:07 rh1 echo[11487]: hallo world
Jan 19 15:48:07 rh1 systemd[1]: run-ref10f3923f304b379bfcf732e79d2987.service: Deactivated successfully.
```

On the one hand, this is great. Built-in-logging for the scripts! On the other hand, it's also again that if we spawn too many scripts, it will spam the logs in a bad way.